### ISSUE TITLE
Keyboard navigation with CompoundColumns

### ISSUE THREAD

**Issue by ghost** [SitePen/dgrid#994]
Keyboard navigation (using arrows) in header of dgrid with CompoundColumns extension does not work properly. 

It is possible to navigate to first invisible row (.dgrid-spacer-row) and after pressing the SPACE, the row becomes visible with sort arrow in one of its cells (see attached image).

The second issue is that navigation between all visible header cell is not possible (e.g. when I focus header 'Middle', I can navigate using arrows only to headers 'First' and 'Last').

Both issues can be simulated in the first grid in CompoundColumn test (dgrid/test/extensions/CompoundColumns.html).

![image](https://cloud.githubusercontent.com/assets/1631199/4147174/ec1f1434-3414-11e4-9e4d-d832bfc3ccc8.png)
